tests/manage/z_cluster/nodes/test_nodes_restart.py

```diff
@@ -298,3 +300,27 @@ def test_pv_provisioning_under_degraded_state_stop_rook_operator_pod_node(
 
         # Checking cluster and Ceph health
         self.sanity_helpers.health_check()
+
+    @bugzilla('1873938')
+    @skipif_no_lso
+    def test_pv_after_reboot_node(self, nodes):
+        """
+        Verify unexpected PV is not created after node reboot on LSO cluster
+
+        """
+        pv_before_reset = get_pv_names()
+        worker_node = get_typed_nodes(
+            node_type=constants.WORKER_MACHINE, num_of_nodes=1
+        )
+        nodes.restart_nodes(nodes=worker_node, wait=False)
```

Comment:
Here after initiating node reboot, the node function will not wait for the OCP node to reach ready state since you are setting the wait flag to False. 
Is that intentional?
If yes, may be will have to handle wait for the ocp node to reach ready state below.
Please check

Reply:
@PrasadDesala 
I copied from `test_rolling_nodes_restart`.
On `teardown`, we call to `restart_nodes_by_stop_and_start_teardown` function, to make sure all VMs are up by the end of the test.


@ebenahar 
what do you think?

Reply:
In test_rolling_nodes_restart, once we restart the node we are doing sanity_helpers.health_check where it checks for the node to reach ready state. So, this is handled in that test.
Here, we are not checking that the node has reached status reach ready. Immediate after initiating node restart, you are getting the pvs. There can be a chance that the pv might get created after the node has reached ready state, so IMO better we wait till the node has reached ready state and then get the pvs.